### DISCUSSION TITLE
Is there a way to have `Private` area of the note that does not get published?

### DISCUSSION THREAD

**Discussion by nickolay-kondratyev** [dendronhq/dendron#2667] [Q&A]
I thought there was a way to have `Private` area of a note that by default does not get published?

Something like:
```
## Regular-1

## Regular-2

## Private 
...
```

Where the regular headers would get published while the private area would NOT get published. Does this feature exist? 

If not would be awesome to have a tag which would allow certain headers to be removed from published like: 

```
## Regular-1

## Regular-2

## Team only #private 
...
```

**Comment by kevinslin** [comment]
right now we don't. the tricky part about this is that if you mark a part as private in a note, if you publish the repo, the note in the repo will still be public. it is something we are considering if enough people have a use case. the alternative right now is to have the team only part be in a note ref of a note that is in a private note

**Reply by nickolay-kondratyev** [reply]
Not following the part of the note being public in the repo. Do you mean the raw contents of the note are exposed somehow? 

The proposed alternative has a massive downside: refactors done in `vault1` do NOT propagate to `vault2` even if both are in the same workspace at the time of refactor. Hence, to continue leaning into `Refactoring` capabilities of Dendron a single primary vault is required to be utilized.

**Reply by Tycholiz** [reply]
I think he means that while the note won't appear in your Published Dendron Site, it will still appear in your repository. 

I never really saw this as an issue, since I keep my repos private anyway, but I can see the concern. It might give people a false sense of security into what they are marking as "private"
